packages/oidc-client/src/OidcClientModule.ts

```diff
@@ -0,0 +1,25 @@
+import type { DependencyManager, Module } from '@aries-framework/core'
+
+import { W3cCredentialService } from '@aries-framework/core'
+
+import { OidcClientApi } from './OidcClientApi'
+import { OidcClientService } from './OidcClientService'
+
+/**
+ * @public
+ */
+export class OidcClientModule implements Module {
+  public readonly api = OidcClientApi
+
+  /**
+   * Registers the dependencies of the question answer module on the dependency manager.
+   */
+  public register(dependencyManager: DependencyManager) {
+    // Api
+    dependencyManager.registerContextScoped(OidcClientApi)
+
+    // Services
+    dependencyManager.registerSingleton(OidcClientService)
+    dependencyManager.registerSingleton(W3cCredentialService)
```

Comment:
The W3cCredentialService is already registered by core. Were you running into issues if it isn't registered?

Reply:
Oh I assumed by calling `registerSingleton` it will make sure it only gets included once if it has already been added. I can remove it, just wasn't sure.

Reply:
It won't cause Issues I think, but if you register the W3cCredentialService, you should also register the W3cCredentialRepository, etc.. Otherwise it will still break if the W3c module isn't registered

Reply:
I have removed it for now. Will add it again if I run into issues down the line.